marketplacetests/tests/test_marketplace_login_during_purchase.py

```diff
@@ -26,7 +30,7 @@ def test_purchase_app(self):
 
         details_page = marketplace.navigate_to_app(APP_NAME)
         ff_accounts = details_page.tap_purchase_button(is_logged_in=False)
-        ff_accounts.login(acct.email, acct.password)
+        ff_accounts.login(username, password)
```

Comment:
I changed this to the credentials user because the create/confirm window appears if we login with a new user